src/pages/console/formbuilder/data-binding.mdx

```diff
@@ -0,0 +1,78 @@
+export const meta = {
+  title: 'Data binding',
+  description:
+    'Cloud connected forms can be bound to data models with relationships, allowing multiple data models to be updated upon submission.'
+};
+
+## Connected forms
+Connected Forms are bound to a model in your data schema. Whenever a connected form is submitted, a record is created or updated in the bound data model automatically, with some or all of the form's input fields mapping to fields in the data model. Connected forms work automatically - no `onSubmit` handling is required.
+
+## Unconnected forms
+Unconnected Forms are standalone React components that can be used in any React or Nextjs project, even without an AWS account. Upon submission, the input values for the form are [accessible via the `onSubmit` hook for handling](/console/formbuilder/lifecycle/#handle-form-data-submissions---onsubmit). 
+
+You can build unconnected forms without ever logging into AWS [using the Amplify Sandbox](https://sandbox.amplifyapp.com/ui-library). 
+
+## Types of forms
+
+All connected and unconnected forms are either a **Create** form or an **Update** form. 
+
+### Create forms
+
+Create forms render a form with empty inputs. If a create form is connected to a data model, will always generate a new record upon submission.
+
+### Update forms
+Update forms expect an input value in order to pre-populate the form. 
+
+For update forms that are connected to a data model, you can use the `id` prop, or the model prop:
+* `id` prop: id string of the record you want to update. For example:
+
+```jsx
+<AuthorUpdateForm id='ac74af5c-3aab-4274-8f41-23e1e6576af5' />
+```
+
+* Model prop: if your form is bound to a data model named `Author`, your form will have a prop named `author` as well, which can receive a record. For example:
+
+```jsx
+
+<AuthorUpdateForm author={authorRecord}>
+
+```
+It is generally recommended to use the model prop instead of the `id` prop. 
+
+For unconnected update forms, you can pass an object to the `initialData` prop to pre-populate the form. However, as with all unconnected forms, you must handle the form submission in code. 
+
+## Customizing data binding
+
+### Connect an unconnected form
```

Comment:
this sounds a little off. maybe something along the lines of `connect to a data model` or `Map to a data model`